apps/console/src/features/validation/pages/validation-config-edit.tsx

```diff
@@ -0,0 +1,769 @@
+                "data-testid": `${ componentId }-page-back-button`,
+                onClick: handleBackButtonClick,
+                text: isApplicationRedirect ?
+                    t("console:manage.features.validation.goBackToApplication")
+                    : t("console:manage.features.validation.goBackToValidationConfig")
+            } }
+            bottomMargin={ false }
+            contentTopMargin={ true }
+            pageHeaderMaxWidth={ true }
+        >
+            <Ref innerRef={ pageContextRef }>
+                <Grid
+                    className="mt-3"
+                >
+                    <Grid.Row columns={ 1 }>
+                        <Grid.Column width={ 16 }>
+                            <EmphasizedSegment className="form-wrapper" padded={ "very" }>
+                                <div className="validation-configurations password-validation-configurations">
+                                    { (!isValidationLoading && !isLoading)  ?
+                                        (<Form
+                                            id={ FORM_ID }
+                                            initialValues={ initialFormValues }
+                                            uncontrolledForm={ true }
+                                            validate={ null }
+                                            onSubmit={
+                                                (values: ValidationFormInterface) => handleUpdateMyAccountData(values)
+                                            }
+                                            enableReInitialize={ true }
+                                        >
+                                            { /*<Field.Radio*/ }
+                                            { /*    key={ "validationType" }*/ }
+                                            { /*    ariaLabel={ "rulesEnabled" }*/ }
+                                            { /*    name={ "validationType" }*/ }
+                                            { /*    label={ "Rules" }*/ }
+                                            { /*    required={ false }*/ }
+                                            { /*    checked={ isRuleType }*/ }
+                                            { /*    listen={ (value: string) => {*/ }
+                                            { /*        setRuleType(true);*/ }
+                                            { /*    } }*/ }
+                                            { /*    width={ 16 }*/ }
+                                            { /*    data-testid={ `${ componentId }-rules-toggle` }*/ }
+                                            { /*/>*/ }
+                                            { isRuleType &&
+                                                (<div className="validation-configurations-form">
+                                                    <div className="criteria" >
+                                                        <label>Must be between</label>
```

Comment:
See if Semantic has a `Form.Label`
```suggestion
                                                        <label for="minLength">Must be between</label>
```